lib/gnat.ex

```diff
@@ -25,14 +25,18 @@ defmodule Gnat do
   * `sid` - The subscription ID corresponding to this message. You generally won't need to use this value directly.
   * `reply_to` - A topic supplied for expected replies
   * `headers` - A set of NATS message headers on the message
+  * `status` - Similar to an HTTP status, this is present for messages with headers and can indicate the specific purpose of a message. Example `status: "408"`
```

Comment:
Hmmmm I can find several examples in the ADRs, but I don't think we can provide an exhaustive list since clients can publish their own messages with headers and provide their own status codes